game-core/src/main/java/games/strategy/triplea/delegate/data/MoveDescription.java

```diff
@@ -13,17 +12,17 @@
 public class MoveDescription extends AbstractMoveDescription {
   private static final long serialVersionUID = 2199608152808948043L;
   private final Route route;
-  private final Collection<Unit> transportsThatCanBeLoaded;
+  private final Map<Unit, Unit> unitsToTransports;
   private final Map<Unit, Collection<Unit>> dependentUnits;
 
   public MoveDescription(
       final Collection<Unit> units,
       final Route route,
-      final Collection<Unit> transportsThatCanBeLoaded,
+      final Map<Unit, Unit> unitsToTransports,
       final Map<Unit, Collection<Unit>> dependentUnits) {
```

Comment:
Not your fault, but we try to annotate nullable fields and parameters with `@Nullable` in new code/refactored code, so it would be nice if you could do the same when you explicity pass null somewhere